dev-lang/nasm/nasm-2.14.02.recipe

```diff
@@ -1,39 +1,48 @@
-SUMMARY="The Nasm assembler"
+SUMMARY="The NASM assembler"
 DESCRIPTION="The Netwide Assembler, NASM, is an 80x86 and x86-64 assembler \
-designed for portability and modularity. It supports a range of object file
+designed for portability and modularity. It supports a range of object file \
 formats, including Linux and *BSD a.out, ELF, COFF, Mach-O, Microsoft 16-bit \
 OBJ, Win32 and Win64. It will also output plain binary files. Its syntax is \
 designed to be simple and easy to understand, similar to Intel's but less \
 complex. It supports all currently known x86 architectural extensions, and \
 has strong support for macros."
 HOMEPAGE="https://www.nasm.us/"
-COPYRIGHT="1996-2016 NASM Authors"
+COPYRIGHT="1996-2018 The NASM Authors"
 LICENSE="BSD (2-clause)"
 REVISION="1"
 SOURCE_URI="https://www.nasm.us/pub/nasm/releasebuilds/$portVersion/nasm-$portVersion.tar.bz2"
-CHECKSUM_SHA256="00b0891c678c065446ca59bcee64719d0096d54d6886e6e472aeee2e170ae324"
-PATCHES="nasm-$portVersion.patchset"
+CHECKSUM_SHA256="34fd26c70a277a9fdd54cb5ecf389badedaf48047b269d1008fbc819b24e80bc"
+#PATCHES="nasm-$portVersion.patchset"
 
-ARCHITECTURES="x86_gcc2 x86 x86_64"
+commandBinDir=$binDir
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
```

Comment:
Check ordering from the guidelines: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering
architectures before commandbindir ...

Reply:
Sorry, put that in the wrong place :(.